config/schema.json

```diff
@@ -100,6 +111,18 @@
     "rights_holder": {
       "type": "text"
     },
+    "rights_statement": {
+      "type": "nested",
+      "properties": {
+        "uri": {
+          "type": "keyword"
+        },
+        "label": {
+          "type": "text",
+          "copy_to": ["full_text", "rights_statement_facet"]
```

Comment:
It's not a big deal but I think I would leave it out of the full_text field and just copy to the facet...